prow/plugins/sigmention/sigmention.go

```diff
@@ -85,7 +85,8 @@ func handle(gc githubClient, log *logrus.Entry, e *github.GenericCommentEvent) e
 		return nil
 	}
 
-	sigMatches := sigMatcher.FindAllStringSubmatch(e.Body, -1)
+	re := fmt.Sprintf(`(?m)@%s/sig-([\w-]*)-(misc|test-failures|bugs|feature-requests|proposals|pr-reviews|api-reviews)`, owningOrg)
+	sigMatches := regexp.MustCompile(re).FindAllStringSubmatch(e.Body, -1)
```

Comment:
`MustCompile` can panic which would kill the entire hook binary. Please use `Compile` instead and handle any errors.

Reply:
Or add that compile check to the config loading unit test so we can't check in broken strings here?

Reply:
done